airbyte-ci/connectors/connectors_qa/src/connectors_qa/models.py

```diff
@@ -21,6 +21,8 @@
 
 ALL_TYPES = ["source", "destination"]
 
+ALL_SUPPORT_LEVELS = ["certified", "community"]
```

Comment:
nit: if we add a new support level we would have to update this list.
We could make `applies_to_connector_support_levels` return `None` by default which would me "applies to all support levels"